### ISSUE TITLE
Consider removing outdated docs

### ISSUE THREAD

**Issue by artur-beljajev** [internetee/registry#247]
https://github.com/internetee/registry/blob/master/doc/models_complete.svg
https://github.com/internetee/registry/blob/master/doc/controllers_complete.svg

**Comment by vohmar** [comment]
@artur-beljajev no, these docs need to be updated, not removed

**Comment by artur-beljajev** [comment]
Why do we need them?

**Comment by vohmar** [comment]
it explains how the system works - data models have been a big help in the past. Documentation is good in case things are not self-explanatory.

**Comment by artur-beljajev** [comment]
ok, then we probably need to automate this process somehow, since above mentioned docs can get outdated extremely easily (as they are now).

**Comment by vohmar** [comment]
yes, please do. These schemas were automatically generated by the initial developers

**Comment by artur-beljajev** [comment]
I submit removal of № 1, 2. Concerning № 3, they give no benefit, IMHO. They don't explain _how_ the system works, they just provide raw data of database columns and model relationship, which are anyway available by just looking at DB or models directly. Instead of investing time to support docs, I would invest this time in making the code more understandable.

If you still prefer to automate this, I kindly ask to explain the benefits we gain if we do this.

**Comment by vohmar** [comment]
no1 might be necessary for modular architecture
no2 seems necessary as it guides the setup of the autotest environment.  
no3 db complete model must be kept updated
no4 update as necessary

**Comment by artur-beljajev** [comment]
```
All registry demo data can be found at:
db/seeds.rb
```
is also outdated

**Comment by vohmar** [comment]
just as a reminder we need to update these documents in addition to other doc existing and mentioned in this ticket (added links to rejected PRs as reference):

* database diagrams (https://github.com/internetee/registry/pull/1073)
* mod-epp doc needs to be replaced with epp-proxy doc (https://github.com/internetee/registry/pull/1284)
* autotest setup (https://github.com/internetee/registry/pull/1285)

there are gems used to update documentation previously. Some of these gems are listed here:
* https://github.com/internetee/registry/pull/1358/files

**Comment by yulgolem** [comment]
2, 4 updated by https://github.com/internetee/registry/pull/1284

**Comment by yulgolem** [comment]
3 there is no need to store in project, RubyMine generates them on the fly (https://www.jetbrains.com/help/ruby/working-with-diagrams.html#class_diagram).
